src/hooks/useENSRegistrationForm.ts

```diff
@@ -141,8 +141,6 @@ export default function useENSRegistrationForm({
         };
       }, {});
       updateRecords(records);
-    } else if (mode === REGISTRATION_MODES.EDIT && !isEmpty(defaultRecords)) {
-      updateRecords(defaultRecords);
```

Comment:
This is a redundant call.

We already set records in edit mode here:
https://github.com/rainbow-me/rainbow/blob/%40jxom/fix-edit-default-state/src/hooks/useENSModifiedRegistration.ts#L85-L95